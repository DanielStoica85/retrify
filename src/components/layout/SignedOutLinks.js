import React from 'react';
import { NavLink, NavItem } from 'reactstrap';

const SignedOutLinks = () => (
    <React.Fragment>
        <NavItem>
            <NavLink href="/register">Sign Up</NavLink>
        </NavItem>
        <NavItem>
            <NavLink href="/login">Login</NavLink>
        </NavItem>
    </React.Fragment>
);

export default SignedOutLinks;
