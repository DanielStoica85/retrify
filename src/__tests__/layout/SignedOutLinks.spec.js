import React from 'react';
import renderer from 'react-test-renderer';
import SignedOutLinks from '../../components/layout/SignedOutLinks';

describe('SignedOutLinks component', () => {
    it('renders correctly', () => {
        const component = renderer.create(<SignedOutLinks />).toJSON();
        expect(component).toMatchSnapshot();
    });
});
