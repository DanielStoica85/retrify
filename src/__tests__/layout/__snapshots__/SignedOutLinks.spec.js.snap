// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SignedOutLinks component renders correctly 1`] = `
Array [
  <li
    className="nav-item"
  >
    <a
      className="nav-link"
      href="/register"
      onClick={[Function]}
    >
      Sign Up
    </a>
  </li>,
  <li
    className="nav-item"
  >
    <a
      className="nav-link"
      href="/login"
      onClick={[Function]}
    >
      Login
    </a>
  </li>,
]
`;
